Classes/Module/Statistics.php

```diff
@@ -1599,6 +1733,7 @@ public function makeStatTempTableContent(array $mrow)
                     break;
                 case '1':
                     // treat html links like plain text
+                    break;
```

Comment:
there must be no break for "1". it's intended, that response_type 1 and 2 will be handled the same way